src/pages/scripting/samples/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Samples
+description: UXP Script samples
+contributors:
+  - https://github.com/amandahuarng
+---
+
+# Samples
+
+## Create a new layer 
+You can create a script that creates a new layer, with the help of the Photoshop DOM API: 
+```js
+const app = require('photoshop').app;
+await app.documents.add();
+await app.activeDocument.createLayer();
+app.activeDocument.layers[0].name = 'New layer';
+```
+
+## Access the local filesystem
+You can create a script that accesses your local filesystem, with the help of the UXP storage module: 
+```js
+const uxpfs = require("uxp").storage;
+const ufs = uxpfs.localFileSystem;
+
+await ufs.getTemporaryFolder().
+    then((result) => {
+        return result.getMetadata();
+    }).then((result) => {
+        console.log(JSON.stringify(result));
+    }).catch((error) => {
+        console.log("Local File system Error: " + error);
+});
+```
+
+## Using a file picker
```

Comment:
@amandahuarng can you also add an example explaining 'global await'?

```
function anyAsyncFunction() {
    return new Promise((resolve, reject) => {
        const val = Math.random();
        if (val > 0.5) {
            console.log("Resolve promise");
            resolve(val);
        } else {
            console.error("Rejecting promise");
            reject(val);
        }
    })
}

// Global await
try {
    const val = await anyAsyncFunction();
    console.log("Value from function", val);
} catch {
    console.log("Rejected promise!;")
}
console.log("Script completed");
```

Reply:
This is already covered in the "How it works" section. @padmkris123

Reply:
should I call it "top-level await"? @samgannawayA @padmkris123  I can change it

Reply:
The top-level-await is generally used in the context of JS modules, which UXP doesnt support yet. Can we keep it as global await itself?